src/patient-search/patient-search.component.tsx

```diff
@@ -87,7 +89,7 @@ export default function PatientSearch(props: PatientSearchProps) {
       <div className={styles.patientSearchHeader}>
         <input
           className={`omrs-type-title-5 ${styles.patientSearchInput}`}
-          placeholder="Search for patient"
+          placeholder={`${t("Search for patient", "Search for patient")}`}
```

Comment:
Do we need to wrap it in a template string? Would `t("Search for patient", "Search for patient")` be sufficient, too?